gravitee-am-repository/gravitee-am-repository-jdbc/src/main/java/io/gravitee/am/repository/jdbc/common/dialect/AbstractDialectHelper.java

```diff
@@ -172,6 +172,8 @@ protected final String convertFieldName(FilterCriteria criteria) {
                 return "enabled";
             case "emails.value":
                 return "email";
+            case "displayName":
+                return "name";
```

Comment:
**issue**: User has name and displayName, with this addition you will break the user filtering IMO. Can you check that point please ?

Reply:
It breaks, had to create a separate method to convert Groups name